index.js

```diff
@@ -0,0 +1,12 @@
+const { runner } = require('hygen');
+const Logger = require('hygen/lib/logger');
+const path = require('path')
+const defaultTemplates = path.join(__dirname, './_templates');
+
+module.exports = () => runner('component with-prompt', {
+    templates: defaultTemplates,
+    cwd: process.cwd(),
+    logger: new Logger(console.log.bind(console)),
+    debug: !!process.env.DEBUG,
```

Comment:
Why `!!`? Shouldn't that be the same as without any of these exclamation marks?

Reply:
I actually prefer `!!` but that's because I'm used to `typescript` explicitly converting a value explicitly to `boolean`. I'm good either way.